doc/source/conf.py

```diff
@@ -507,8 +507,7 @@ def _get_c_source_file(obj):
         # todo: come up with a better way to generate these
         return None
 
-
```

Comment:
If you're editing again, just keep the empty line in-place, so this doesn't change.